go.mod

```diff
@@ -1,11 +1,14 @@
 module github.com/alpacahq/alpacadecimal
 
-go 1.18
+go 1.23
```

Comment:
Overall LGTM! but little hesitation to bump required go version just because of benchmarking even though golang only supports latest two versions.

Reply:
Yeah, I agree. Is it possible to use different version for benchmarks only?